Makefile

```diff
@@ -10,22 +10,18 @@ endif
 .PHONY: all
 all: test
 
-vendor:
-	glide install
-
 .PHONY: test
-test: vendor
+test:
 	go test $(V) ./... -race
 
 .PHONY: generate
-generate: vendor
+generate:
 	GENERATE_FIXTURES=true $(MAKE) test
```

Comment:
If you want to continue yak shaving this, you can avoid the extra make invocation by doing this:

```Makefile
.PHONY: generate
generate: export GENERATE_FIXTURES=true
generate: test
```